### PR TITLE
Missing mono lemmas

### PR DESCRIPTION
**Pull request by CohenCyril**
##### Motivation for this change

Added lemmas about monotony of functions `nat -> T` where `T` is an ordered type

##### Things done/to do

- [x] added corresponding entries in `CHANGELOG_UNRELEASED.md`
- ~added corresponding documentation in the headers~

##### Automatic note to reviewers

Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-following,-reviewing-and-playing-with-a-PR#checklist-for-reviewing-a-pr) and make sure there is a milestone.

### CODE REVIEW

File: CHANGELOG_UNRELEASED.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+- Added lemmas about monotony of functions `nat -> T` where `T` is an
+  ordered type and `homo_ltn_lt_in`, `incn_inP`, `nondecn_inP`,
```
Comment (ybertot) [inline CHANGELOG_UNRELEASED.md:12]: grammar mistake here.

Comment (CohenCyril) [reply CHANGELOG_UNRELEASED.md]: done?

Comment (ybertot) [reply CHANGELOG_UNRELEASED.md]: Good for me!

**Comment by affeldt-aist** [comment]
@CohenCyril Is it ok to merge or do you want to squash it before?

**Comment by CohenCyril** [comment]
you can hit the squash a merge button

**Pull request merged**